### PR TITLE
View adoption form

### PR DESCRIPTION
**Pull request by Krabby-Patties**
added way to view adoption forms

### CODE REVIEW

File: src/components/ViewAdoptionForms.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        setNewApproval(event.currentTarget.value)
+    }
+
+    const handleSelectionChange = event => {
+        setCurrentlySelectedForm(JSON.parse(event.currentTarget.value))
+    }
+
+    const handleApprovalSubmit = event => {
+        event.preventDefault()
+        //fetch request goes here:
```
Comment (DanAlberts) [inline src/components/ViewAdoptionForms.js]: Unnecessary comment here.  Please remove.

**Review by DanAlberts — APPROVED** [review]

**Review by DanAlberts — APPROVED** [review]

File: src/components/ViewAdoptionForms.js (hunk 1 above)
Comment (DanAlberts) [inline src/components/ViewAdoptionForms.js]: Not sure why its refusing to let me request changes.

Comment (DanAlberts) [reply src/components/ViewAdoptionForms.js]: Please fix the above!

**Review by DanAlberts — CHANGES_REQUESTED** [review]

File: src/components/ViewAdoptionForms.js (hunk 1 above)
Comment (Krabby-Patties) [reply src/components/ViewAdoptionForms.js]: deleted it, thank you

**Review by DanAlberts — APPROVED** [review]

**Pull request merged**